content/docs/operating/security.md

```diff
@@ -19,6 +19,11 @@ prometheus-team@googlegroups.com. We will fix the issue and coordinate a
 release date with you, acknowledging your effort and mentioning you by name
 if you want.
 
+Prometheus is maintained by volunteers, not by a company. Therefore, fixing
+security issues is done on a best-effort basis. We strive to release security
+fixes within 7 days for: Prometheus, Alertmanager, Node_exporter,
```

Comment:
I'd either say "node_exporter" or "Node Exporter" (probably the latter in a normal English sentence), but mixing snake_case with CamelCase seems odd (same for Blackbox Exporter).

Reply:
ack